src/formatDate.js

```diff
@@ -50,7 +50,60 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let year = '';
+  let month = '';
+  let day = '';
+  const [,,, separatorFF] = fromFormat;
```

Comment:
It's easier to read, in my opinion. But it's really good that you know how to use destructuring
```suggestion
  const separator = fromFormat[3];
```